beacon_node/eth2-libp2p/src/rpc/handler.rs

```diff
@@ -393,7 +393,7 @@ where
                 let err = HandlerErr::Inbound {
                     id: inbound_id,
                     proto: *protocol,
-                    error: RPCError::ErrorResponse(*code, reason.clone()),
+                    error: RPCError::ErrorResponse(*code, reason.to_string()),
```

Comment:
@AgeManning @pawanjay176 I had to redo this change during the merge, I think I got it right, but just wanted to check I didn't miss anything -- I deleted a big chunk of `inject_event` that had been deleted on `master`, and transferred the `to_string` changes that had been added there to this part of the code.

Reply:
Looks correct to me.